src/sentry/services/hybrid_cloud/organization.py

```diff
@@ -25,6 +25,16 @@
 from sentry.silo import SiloMode
 
 
+# This 'assumes' the naming conventions observed in the system thus far, but there is no enforcement currently,
+# except that there would be an attribute exception (hopefully in tests).
+def escape_flag_name(flag_name: str) -> str:
+    return flag_name.replace(":", "__").replace("-", "_")
+
+
+def unescape_flag_name(flag_name: str) -> str:
+    return flag_name.replace("__", ":").replace("_", "-")
```

Comment:
Where do we use the `__` convention for flags?

Reply:
We don't, but as fields on classes I wanted to escape the ":" and "-" we do use.  This function unescapes by returning an "escaped" flag name back to it's original.

Reply:
So my comment is saying, the naming convention currently is that flags use ':' and '-' to namespace flags, which means they must be accessed using '[]'.  Technically I could modify the data classes to match that, but my thinking is that when we go to codegen from RPC, ideally as much of these dataclass attributes that can "just line up" with our IDL, the better.  While it may not always be possible, the ideal world is one in which double serialization is reduced, esp for something like the Organization object.